docs/reference/tab-widgets/snapshot-repo.asciidoc

```diff
@@ -0,0 +1,15 @@
+// tag::cloud[]
+{ess} supports repositories for the following providers:
```

Comment:
Given that this list is a subset of all repository types that the searchable snapshot feature supports, whether someone not reading that this only applies to Elastic Cloud will get the impression that other repo types are not available for the cold/frozen tiers

Reply:
Thanks for pointing this out. To avoid confusion, I've reworded this section to emphasize that the listed repos are ones supported by searchable snapshots.

The content will also appear in an ESS-specific tab. It's hard to tell in the source, but it's a bit clearer in the preview: 
https://elasticsearch_71195.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/use-elasticsearch-for-time-series-data.html